live-examples/js-examples/expressions-commaoperators.html

```diff
@@ -0,0 +1,15 @@
+<pre>
+<code id="static-js">var x = 1;
+
+console.log(x++, x);
+// expected output: 1
```

Comment:
I think this is problematic because in a real console (and this one, if we implement it) that won't be the output. Perhaps we can just omit this bit?

Reply:
Oh interesting - good spot.